.github/workflows/unit-tests.yml

```diff
@@ -88,6 +92,8 @@ jobs:
         include:
           - os: ubuntu-latest
             python_version: '3.9'
+          - os: ubuntu-latest
+            python_version: '3.10'
```

Comment:
Python 3.10 is the latest version Pypy supports, based on https://downloads.python.org/pypy/